migrations/20230926080752_cw721_and_cw20_add_smart_contract_address.ts

```diff
@@ -0,0 +1,60 @@
+import { Knex } from 'knex';
+import CW721Contract from '../src/models/cw721_contract';
+import _ from 'lodash';
+import { Cw20Contract } from '../src/models';
+
+export async function up(knex: Knex): Promise<void> {
+  await knex.schema.alterTable('cw721_contract', (table) => {
+    table.string('smart_contract_address');
+  });
+  await knex.schema.alterTable('cw20_contract', (table) => {
+    table.string('smart_contract_address');
+  });
+  await knex.transaction(async (trx) => {
```

Comment:
hình như cái knex này là sẵn trong transaction rồi đấy

Reply:
em chưa hiểu ý anh đoạn này lắm

Reply:
toàn bộ hàm up này là được bọc trong transaction rồi

Reply:
không phải anh nhé, trước em ko dùng 
**await knex.transaction(async (trx) => {**
nó bị lỗi vì không cùng transaction với các file migration khác

Reply:
https://knexjs.org/guide/migrations.html#transactions-in-migrations

Reply:
em resolve đoạn này rồi anh nhé